### DISCUSSION TITLE
pyqt6 package

### DISCUSSION THREAD

**Discussion by risky777** [termux/termux-packages#19309] [Ideas]
I see that pyqt6 exists for desktop-linux-aarch64, like archlinux-aarch64. This seems to me like indicator that it is possible to have pyqt6 for termux.

Background-info:
- I'm trying to get eric-ide-python on termux-xfce4. Since termux has pyqt5, I got old eric6 installed on termux. But when I start eric6 I get it's splashscreen but it stops with error-message, I found on internet that this error is caused by using old eric6 on python-version newer than 3.10. So, I need pyqt6 to try to jumpstart current eric7 on python-3.11. And, pyqt6 seems like ONLY package I'm missing from desktop-linux-aarch64.

**Comment by TomJo2000** [comment]
This should have probably been opened as a package request issue.
But, let's get this answered.

---

There's already tracker for Qt6
- https://github.com/termux/termux-packages/issues/10854

Though the effort seems to have stalled,
the major issue is that packaging Qt is a major pain in the ass.
We *should* get around to it, but nobody has had the time
to package and patch the dozens of Qt6 packages
which have currently packaged Qt5 equivalents.
PRs are always welcome, but this is unfortunately a major rabbit hole with no quick solution.

**Reply by risky777** [reply]
I tried to open it as package request, but there were questions for which I didn't have answers, so I opened it here.

Thanks for fast reply. I hoped I could play with python-kivy directly on android tablet, but since there is no real python ide available (not only eric but any) I guess that is impossible.

**Reply by TomJo2000** [reply]
You can write Python in any editor of your choice (`neovim`/`vim`, `emacs`, `nano`, `micro`, etc.),
and most of the extensible ones will have Python integration plugins available,
though how you prefer to set up your personal development environment is obviously your own preference.
